app/controllers/couches_controller.rb

```diff
@@ -0,0 +1,33 @@
+class CouchesController < ApplicationController
```

Comment:
Since these are user couches (i.e we're not listing the couches on their own - we are listing the couches for a particular host), what would you think about following the pattern of app/controllers/users/couches_controller.rb instead of app/controllers/couches_controller.rb ?